plugin/PlayAssetDelivery/src/main/java/com/google/play/core/godot/assetpacks/PlayAssetDelivery.java

```diff
@@ -48,7 +48,6 @@
   private AssetPackManager assetPackManager;
 
   private final String ASSET_PACK_STATE_UPDATED = "assetPackStateUpdated";
-  private final String FETCH_STATE_UPDATED = "fetchStateUpdated";
```

Comment:
Removed this signal since we are only using the global assetPackStateUpdated listener.